### ISSUE TITLE
Is there any subcategory for scene? // cosineannealinglr 

### ISSUE THREAD

**Issue by lliee1** [zwx8981/LIQE#8]
Hi. I'm currently using LIQE for custom dataset.
Thank you for your code and paper.

Because custom dataset do not have any scene and dists label, but have caption, i make scene and dists label using pretrained CLIP text encoder. 
I just use cosine similarity about [caption] and [scene,dists category].

because there is gap between scene categories and my caption of custom dataset, there are so many 'others' in my scene label. so there is class imbalance.
Fortunately, dists has subcategory that you defined at 'train_unique_clip_weight.py'. and there is little 'other' in dists label that i make.

so, Is there any subcategory for scene?? If exists, I can use subcategory and mapping like distortion map that you did. 

Another question is scheduler.
I also use cosineannealing for scheduler that you did. and eta_min = 0 for your scheduler.

1. can model be trained at learning_rate approximately same with 0? 
2. Is there hyperparameter that you use for eta_min? 

Thank you.

**Comment by zwx8981** [comment]
@lliee1 Hi, thanks for your interest in our work!

For your first question, unfortunately, we don't have such subcatgories of scenes. If it is meaningful to train the model on your custom dataset with the scene information, I would suggest using predefined categories in object recognition (e.g., ImageNet) or scene classification datasets (e.g., Place365). By doing so, you can apply the pretrained classifier to automatically annotate your custom dataset.

For your second question, I think the good hyper-parameters should be tuned task by task, so I would suggest just try more options, and pick the best one according to the performance on the validation set.
